src/components/fields/Radio.js

```diff
@@ -1,61 +1,34 @@
 import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-import Field from '../Field';
+import * as PropTypes from 'prop-types';
+import { RadioGroupContext } from '../FormContext';
 
-class Radio extends Component {
+export default class Radio extends Component {
```

Comment:
Backward compatibility is must. We won't change major version for each update. So please leave `Radio` components and create a new one (sm. like  `RadioGroupItem`).